package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ultimaker/eslint-config",
-    "version": "1.0.0",
+    "version": "0.0.0",
```

Comment:
Uhm?

Reply:
Since we didn't have the release mechanism working yet on GitLab, we ended up bumping the version with nothing getting published. I deleted the tag and unbumped the version so that now that the CI system is running I can publish 1.0.0 successfully.